internal/command/meta_backend.go

```diff
@@ -1025,11 +1025,24 @@ func (m *Meta) backend_C_r_s(c *configs.Backend, cHash int, sMgr *clistate.Local
 		Hash:      uint64(cHash),
 	}
 
-	// Verify that selected workspace exist. Otherwise prompt user to create one
+	// Verify that selected workspace exists in the backend.
 	if opts.Init && b != nil {
-		if err := m.selectWorkspace(b); err != nil {
+		err := m.selectWorkspace(b)
+		if err != nil {
 			diags = diags.Append(err)
-			return nil, diags
+
+			// HACK: A compatibility oddity with the 'remote' backend.
```

Comment:
We're just starting to establish a convention of explicitly using `FIXME` to mark technical debt, and that seems fairly appropriate here instead of `HACK`, if you wouldn't mind changing it 🙏